### PR TITLE
Allow width and height attributes for table, TD and TH elements

### PR DESCRIPTION
**Pull request by cherrypj**

### CODE REVIEW

File: services/drupal/config/sync/filter.format.filtered_html.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ filters:
...
...<summary> <form action method> <fieldset disabled form name> <legend> <label for form> <input checked size type value disabled placeholder required readonly autocomplete autofocus> <textarea rows cols disabled placeholder required readonly autocomplete autofocus> <meter min max low high optimum form value> <select multiple> <optgroup disabled label> <option value> <output for form name> <button type value> <datalist> <keygen autofocus challenge disabled form keytype name> <progress max value> <img alt crossorigin height longdesc src width usemap data-caption data-align> <embed height src type width> <object form height name type usemap width> <param name value> <iframe allowfullscreen height name sandbox scrolling seamless src srcdoc width> <video autoplay buffered controls crossorigin height loop muted played preload poster src width> <audio autoplay buffered controls loop muted played preloadsrc> <source media src type> <track default kind label src srclang> <canvas height width> <map name> <area alt coords href hreflang media rel shape target type> <drupal-media data-caption data-align data-entity-type data-entity-uuid alt><drupal-inline-media data-entity-type data-entity-uuid>'
```
Comment (azinck) [inline services/drupal/config/sync/filter.format.filtered_html.yml:37]: @cherrypj Did you intentionally remove the aria-* attribute from all tags?

Comment (cherrypj) [reply services/drupal/config/sync/filter.format.filtered_html.yml:37]: Ohhhh. So that's why this work didn't land in live. I will fix.

**Comment by azinck** [comment]
Closing this because it conflicts with another PR.  I fixed this in a separate commit.

**Comment by cherrypj** [comment]
@azinck I think in the separate commit, you missed adding the `height` and `width` attributes to `th` elements.

**Pull request closed without merging**